lib/Sabberworm/CSS/Value/Value.php

```diff
@@ -63,16 +63,25 @@ public static function parseValue(ParserState $oParserState, $aListDelimiters =
 	}
 
 	public static function parseIdentifierOrFunction(ParserState $oParserState, $bIgnoreCase = false) {
-		$sResult = $oParserState->parseIdentifier($bIgnoreCase);
+		$oParserState->setAnchor();
+		$mResult = $oParserState->parseIdentifier($bIgnoreCase);
 
 		if ($oParserState->comes('(')) {
-			$oParserState->consume('(');
-			$aArguments = Value::parseValue($oParserState, array('=', ' ', ','));
-			$sResult = new CSSFunction($sResult, $aArguments, ',', $oParserState->currentLine());
-			$oParserState->consume(')');
+			if ($oParserState->streql('url', $mResult)) {
+				$oParserState->backtrackToAnchor();
+				$mResult = URL::parse($oParserState);
```

Comment:
Do we use `URL::parse` and `CalcFunction::parse` also in other places? Otherwise we could just change them to not parse the function name (or pass it as argument in case of `CalcFunction`).

Also maybe we should just add a `CSSFunction::parse` method so that all three cases have the same structure.